### DISCUSSION TITLE
How to connect Pocket erls TX to s.port

### DISCUSSION THREAD

**Discussion by dangerousvasil** [ExpressLRS/ExpressLRS#2627] [General]
Hello friends
![IMG_20240401_091626](https://github.com/ExpressLRS/ExpressLRS/assets/9209807/ab42611d-5c9d-422c-ae5a-1ce5df9724d9)
 what i wannna do

on erls module i have pinout 
![IMG_20240321_000403](https://github.com/ExpressLRS/ExpressLRS/assets/9209807/5bdaca57-e7e3-4742-9a87-556f72b0f0d7)

on bay i have sbus ground +5v

how i need connect  tx to modul bay?

rx/tx -> sbus 

please help 
i want to understand!

maybe?
module / bay 
tx / sbus 
rx / ground

**Comment by pkendall64** [comment]
Connect the S.PORT pin of the connector to the RX pin only. Let the board go into wifi mode and go to 10.0.0.1/hardware.html
Change serial_tx to be the same value as serial_rx. Go to the bottom and save.

**Comment by dangerousvasil** [comment]
(G)(G)(V)(V)(r?t/x)(s.port)

![photo_2024-04-01 23 14 53](https://github.com/ExpressLRS/ExpressLRS/assets/9209807/727dff91-bb08-4caa-85cc-c4a3834d6391)

**Comment by dangerousvasil** [comment]
![photo_2024-04-01 23 12 49](https://github.com/ExpressLRS/ExpressLRS/assets/9209807/1907bddb-38e8-4b38-ac7c-4ae083a94169)
